### ISSUE TITLE
build failure

### ISSUE THREAD

**Issue by tw4452852** [achilleasa/gopher-os#25]
when I try to make kernel, an error happened when linking:

```
> make
[binutils] checking that installed objcopy version is >= 2.26.0
[nasm] arch/x86_64/asm/cgo_stubs.s
[nasm] arch/x86_64/asm/multiboot_header.s
[nasm] arch/x86_64/asm/rt0_32.s
[nasm] arch/x86_64/asm/data.s
[nasm] arch/x86_64/asm/rt0_64.s
[sed] extracting LMA and VMA from constants.inc
[gcc] pre-processing arch/x86_64/script/linker.ld.in
[go] compiling go sources into a standalone .o file
[objcopy] creating global symbol alias 'kernel.Kmain' for 'github.com/achilleasa/gopher-os/kernel.Kmain' in go.o
[ld] linking kernel-x86_64.bin
build/go.o:(.data+0x50): undefined reference to `_cgo_yield'
make: *** [Makefile:38: build/kernel-x86_64.bin] Error 1
```

Information about go: 

```
> go version
go version devel +1948b7f Wed May 31 10:28:05 2017 +0000 linux/amd64
> go env
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/tw/golib"
GORACE=""
GOROOT="/home/tw/goroot"
GOTOOLDIR="/home/tw/goroot/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build580541135=/tmp/go-build -gno-record-gcc-switches"
CXX="g++"
CGO_ENABLED="1"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
```

**Comment by tw4452852** [comment]
Just find a workaround to fix this.

**Comment by achilleasa** [comment]
Fixed by #26. Thanks for submitting the fix!
